app/helpers/content_substitutor_helper.rb

```diff
@@ -0,0 +1,16 @@
+module ContentSubstitutorHelper
+  def content_substitutor(deployment_environment)
+    @content_substitutor ||= ContentSubstitutor.new(
+      service_name: service.service_name,
+      reference_number_enabled: reference_number_enabled(deployment_environment)
+    )
+  end
+
+  def reference_number_enabled(deployment_environment)
+    ServiceConfiguration.find_by(
+      service_id: service.service_id,
+      deployment_environment: deployment_environment,
+      name: 'REFERENCE_NUMBER'
+    ).present?
+  end
```

Comment:
to get round the need for passing the deployment environment we could use `where`

Reply:
all done!